doc/nrf/app_dev/device_guides/nrf54l/dfu_config.rst

```diff
@@ -0,0 +1,131 @@
+
+1. Navigate to the :file:`zephyr/samples/subsys/mgmt/mcumgr/smp_svr` directory.
+
+#. Build the firmware:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west build -b nrf54l15dk/nrf54l15/cpuapp -p -- -DSB_CONFIG_BOOTLOADER_MCUBOOT=y
+
+#. Flash the firmware onto the device:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west flash
+
+.. note::
+
+  The standalone NSIB configuration is not supported.
+
+NVM protections
+***************
+
+Non-volatile memory protections are crucial for ensuring the integrity and immutability of firmware code.
+These protections are designed to disable read, write, or execute permissions when they are not required.
+There are two types of protection mechanisms:
+
+* NSIB protection - The ``SB_CONFIG_SECURE_BOOT_BOOTCONF_LOCK_WRITES`` Kconfig option enables a lock on NSIB from power-up, which is only removable through a chip erase.
+  This protection uses a single 31kB resistive random access memory controller's (RRAMC) region 3, activated at IC’s power-up if the UICR is programmed accordingly.
+  Enabling this lock, also disables the :kconfig:option:`CONFIG_FPROTECT_ALLOW_COMBINED_REGIONS` Kconfig option, restricting FPROTECT to sizes that are up to 31kB.
+* MCUboot protection - MCUboot can be protected from overwrites using the FPROTECT library.
+  If MCUboot is within 31kB, the device will use RRAMC’s region 4.
+  For sizes exceeding 31kB, both regions 3 and 4 are used, provided MCUboot is configured standalone (without NSIB).
+  To determine protections for sizes between 31kB and 62kB, configure the :kconfig:option:`CONFIG_FPROTECT_ALLOW_COMBINED_REGIONS` Kconfig option.
+
+RAM cleanup
+***********
+
+To prevent data leakage, both NSIB and MCUboot can clear out their RAM upon completion of execution.
+This feature is controlled by the :kconfig:option:`CONFIG_SB_CLEANUP_RAM` Kconfig option.
+
+Supported signatures
+********************
+
+MCUboot accommodates ed25519 and ed25519-pure signatures.
+The latter signature is recommended, but it cannot be used with external memories.
+NSIB supports only the ed25519-pure signature, which is hardcoded.
+
+Signature keys
+**************
+
+:ref:`Key Management Unit (KMU)<ug_nrf54l_developing_basics_kmu>` retains the keys necessary for image signature verification, which must be uploaded simultaneously with the application during the flashing process.
+Currently, encryption keys are not stored in KMU.
+
+.. note::
+   NSIB regenerates its key with each build unless it is specified in the command-line.
+   This could result to unexpected behavior.
+
+Runtime revocation
+******************
+
+.. note::
+   The support for this feature is currently :ref:`experimental <software_maturity>`.
+
+MCUboot can invalidate image verification keys through the ``CONFIG_BOOT_KMU_KEYS_REVOCATION`` Kconfig option.
+You should enable this option if images signed with a compromised key contain critical vulnerabilities.
```

Comment:
call me

Reply:
To make this option available, it should be enabled while building MCUboot. Key removal itself is triggered upon update - when new image is signed with newer key.